engine/src/exec-interface.cpp

```diff
@@ -3330,7 +3330,19 @@ MCControl* MCInterfaceExecCreateControlGetObject(MCExecContext& ctxt, int p_type
 
 void MCInterfaceExecCreateControl(MCExecContext& ctxt, MCStringRef p_new_name, int p_type, MCObject *p_container, bool p_force_invisible)
 {
-    if (MCdefaultstackptr->islocked())
+    
+    MCStack *t_current_stack;
```

Comment:
We try not to leave POD-typed vars (those which do not have a constructor) uninit'd on declaration. I'd suggest:
```
MCStack *t_current_stack = p_container == nullptr ? MCdefaultstackptr : p_container->getstack()
```